pages/01.home/default.md

```diff
@@ -13,4 +13,4 @@ Under the theme of “Bridges to Inclusion,” it endeavors to highlight issues
 
 ## [Keynote Speakers](../02.keynote/about.md)
```

Comment:
@emb03 this one needs to be `## [Keynote Speakers](../02.keynote/keynote.md)` can you fix and do another deploy?